submanifests/zephyr-xenlib.yaml

```diff
@@ -0,0 +1,9 @@
+# Copyright (C) 2024 EPAM Systems.
+#
+# Enable zephyr-xenlib support
+
+manifest:
```

Comment:
Please, provide proper commit message, I am out of context what is happening here and why we need this inside Zephyr

Reply:
It adds  zephyr-xenlib to this particular project. Not sure what to add here :(

https://docs.zephyrproject.org/latest/develop/modules.html#integrate-modules-in-zephyr-build-system

Reply:
dropped